VMEncryption/main/handle.py

```diff
@@ -2103,7 +2116,7 @@ def start_daemon(operation):
 
     #Redirect stdout and stderr to /dev/null.  Otherwise daemon process will
     #throw broken pipe exception when parent process exit.
-    devnull = open(os.devnull, 'w')
+    devnull = open(os.devnull, 'wb')
```

Comment:
added comment:
`#use 'wb' for python3 compat since stdout and stderr are bytes not strings`